src/tools/pub/cmd/pub-get.md

```diff
@@ -145,6 +145,13 @@ By default, pub precompiles executables
 in immediate dependencies (`--precompile`).
 To prevent precompilation, use `--no-precompile`.
 
+### `--enforce-lockfile`
+
+Fetches dependencies, but will refuse to resolve
+if anything deviates from `pubspec.lock` or there is no `pubspec.lock`.
+Such deviations normally trigger a warning, but
+`--enforce-lockfile` turns it into an error.
```

Comment:
**suggestion:** Consider reversing the flow of this sentence to remove the qualifier.

```suggestion
Including `--enforce-lockfile` changes what would
be a warning into an error. This error prevents Dart
from resolving the dependencies until you remove
any deviation from the `pubspec.lock` file.
```

Reply:
Removing deviations from pubspec.lock is not an aspect of this, not going to incoporate